src/ecs/commands/runCommandInContainer.ts

```diff
@@ -122,25 +122,32 @@ export async function runCommandInContainer(
             return
         }
 
-        await runCliCommandInEcsContainer(node, task, command, outputChannel)
+        const execCommand = await node.ecs.executeCommand(node.clusterArn, node.containerName, task, command)
+        const cp = await new ChildProcess(
+            true,
+            'session-manager-plugin',
```

Comment:
Does this work? I'm surprised you don't need the ecs endpoint, and the `Target` JSON passed :

https://github.com/aws/aws-cli/blob/63f3fcf368805d14848769feae4bbf87cc359739/awscli/customizations/ecs/executecommand.py#L105

https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt#L250

Reply:
Yes, this currently is working.

Reply:
We're not passing the ECS endpoint, so this will be going directly to SSM - this is not recommended:

> While starting SSM sessions outside of the execute-command action is possible, this results in the sessions not being logged and being counted against the session limit. 

(from here: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-exec.html)

Reply:
Looks like co-pilot does the 3-arg approach, so we should go with that too, its' simpler and requires fewer service calls ; https://github.com/aws/copilot-cli/blob/ada23de30b1dab0b0d0659a66d45bdfc6646954e/internal/pkg/exec/ssm_plugin.go#L50